src/builder.rs

```diff
@@ -1,150 +0,0 @@
-use crate::{
```

Comment:
Could you explain the reasoning behind the removal of the builder? I think it's a bit too messy to mix building functions in the actual type. Maybe what we should do instead is work on make the capabilities work better with a builder generator like TypedBuilder, but it's probably impossible/not worth it.

Reply:
In my view the Capability struct is like an appliqué over the siwe message, it's sort-of a builder itself. I found when writing that I couldn't distinguish between a builder stage and the recap state. All the operations on it (adding caps and proofs) are commutative, the only validation that needs to happen is checking the contents of the ability strings